vscode-extension/built/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "wireit",
+  "displayName": "wireit",
+  "publisher": "google",
+  "description": "For the wireit script runner",
+  "version": "0.1.0",
+  "contributes": {
+    "commands": [],
+    "jsonValidation": [
+      {
+        "fileMatch": "package.json",
+        "url": "./schema.json"
+      }
+    ]
+  },
+  "main": "./client.js",
+  "activationEvents": [
+    "onLanguage:json"
+  ],
+  "icon": "logo.png",
```

Comment:
(Can't comment on the png).

What happened to the squiggly part of the logo?

Reply:
I cropped it out here, because the main place it's displayed is in a small square in the extension sidebar, and just the text is already crowded there.

Sidebar with different color schemes:

<img width="672" alt="Screen Shot 2022-04-28 at 11 12 41 AM" src="https://user-images.githubusercontent.com/1659/165797607-97baa00c-2dd4-44a7-b584-6c6315cdf269.png">
<img width="675" alt="Screen Shot 2022-04-28 at 11 13 23 AM" src="https://user-images.githubusercontent.com/1659/165797637-3bd8778e-70fe-48ab-9dd7-50f8ee652c4b.png">
<img width="675" alt="Screen Shot 2022-04-28 at 11 14 01 AM" src="https://user-images.githubusercontent.com/1659/165797658-c04a9bf5-ca4b-4de6-9053-6fa9fcdc5632.png">

Full page (after clicking on the extension):
<img width="901" alt="Screen Shot 2022-04-28 at 11 13 01 AM" src="https://user-images.githubusercontent.com/1659/165797728-89e955cc-0493-4972-998c-6e5435b1f34b.png">
<img width="898" alt="Screen Shot 2022-04-28 at 11 13 30 AM" src="https://user-images.githubusercontent.com/1659/165797746-7d912102-8267-44f6-a98e-d6d84a19e012.png">
<img width="905" alt="Screen Shot 2022-04-28 at 11 14 06 AM" src="https://user-images.githubusercontent.com/1659/165797759-c05ea4ab-eff3-48eb-8789-e024c0e50c36.png">

Reply:
Ah ok good call. We could work on an even better suited logo sometime in the future maybe (just the W?). This seems great for now.